content/articles/choosing-wagtail.md

```diff
@@ -0,0 +1,404 @@
+            <b>Responsive design</b>
+            </td>
+            <td>
+            <small>None.</small>
+            </td>
+            <td>
+            <i class="fas fa-lg fa-check-circle" aria-label="yes" style="color: #0c0;"></i> <small>Yes</small>
+            </td>
+        </tr>
+        <tr>
+            <td>
+            <b>Search integration</b>
+            </td>
+            <td>
+            <small>Haystack integration is suggested.</small>
+            </td>
+            <td>
+            <i class="fas fa-lg fa-check-circle" aria-label="yes" style="color: #0c0;"></i> <small>Elasticsearch 6 support is standard. Can also use the database and PostgreSQL search.</small>
+            </td>
+        </tr>
+        <tr>
+            <td>
+            <b>Documentation and community</b>
+            </td>
+            <td colspan="2">
+            <i class="fas fa-lg fa-check-circle" aria-label="yes" style="color: #0c0;"></i> <small>Documentations, support, tutorials and community are robust for both systems.</small>
+            </td>
+        </tr>
+        <tr>
+            <td>
+            <b>Learning curve</b>
+            </td>
+            <td>
+            <small>We didn’t launch a sample to experiment with content providers.</small>
+            </td>
+            <td>
+            <i class="fas fa-lg fa-check-circle" aria-label="yes" style="color: #0c0;"></i> <small>It takes a bit of time in the beginning to set up a model. The complexity or simplicity of the CMS is dependent on how well we design the data the model. We have control over its usability.</small>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+
+## We chose Wagtail CMS
+
+Wagtail meets many, if not, all of the aforementioned criteria for what we want
+from a CMS frameworks. We were persuaded to experiment with Wagtail CMS on the
+forthcoming CTL Portfolio project to see how well it works in a collaborative
+environment. The team—developer, designer, and client (content managers)—worked
+together to draft out user stories, design the website’s UI, and developed the
+data model for the site. The CMS was implemented according to the model, and
+here is an overall look of how the CMS mapped out to the website’s interface.
+
+{{< figure src="/img/assets/wagtail-portfolio.jpg" class="text-center centered responsive" alt="overall look of how the CMS mapped out to the CTL Portfolio interface" >}}
+
+We are pleased with the Wagtail CMS in this project. Some of our (additional)
+shared reasons are:
+
+* Wagtail’s flexible and extendable framework lends itself to product design.
+The designer and the client are not constrained by the CMS to shape the UX for
+the product website.
+* The client, who is also the content manager/editor, can work with the
+developer to design a CMS UI that fits the editor’s needs, instead of trying to
+adjust the needs into an inflexible CMS.
+* Wagtail’s framework provides ample space for the team members to be creative
```

Comment:
I might simplify this to be "Wagtail provides ample space for team members..."

Reply:
Fixed